apps/vanilla/src/pages/blackjack/game/dice.ts

```diff
@@ -0,0 +1,43 @@
+namespace Game {
+
+	/**
+	 * Game dice.
+	 */
+	export class Dice implements ObserverCore.Observer<UI.ThrowButtonClickedEventData> {
+
+		private readonly minValue: number = 1;
+
+		private readonly maxValue: number;
+
+		private readonly diceThrownEventNotifier: ObserverCore.Notifier<Game.DiceThrownEventData> =
+			new ObserverCore.Notifier<Game.DiceThrownEventData>();
```

Comment:
We prefer omitting types if they're properly inferred.
```suggestion
		private readonly diceThrownEventNotifier =
			new ObserverCore.Notifier<Game.DiceThrownEventData>();
```